docs/faq/validator.md

```diff
@@ -5,47 +5,56 @@ description: Frequently asked questions about validation on the camino and colum
 
 # Validator FAQ
 
-## Who can be a Camino validator?
+### Who can be a Camino validator?
 
-Any travel industry market participant who fulfills the following conditions can be a validator on the Camino Mainnet:
+Any participant in the travel industry who meets the following criteria can become a validator on the Camino Mainnet:
 
-- Have at least 100,000 Camino (CAM) tokens locked in their wallet
-- Is KYC / KYB verified
-- Has been admitted as a validator by the validator consortia via a vote
-  The last condition is omitted for Validators who secure the network from the start. To secure the Columbus Testnet, one must only be a market participant in the travel industry.
+- Holds a minimum of 100,000 Camino (CAM) tokens in their wallet.
+- Has undergone and passed a Know-Your-Customer (KYC) and Know-Your-Business (KYB) verification process.
+- Has been elected as a validator by the validator consortium through a voting process.
 
-## Staking Parameters on Camino
+:::note
 
-When a validator is done validating the Camino Mainnet, it receives back the Camino tokens it staked. It may receive a reward for helping to secure the network. A validator only receives a validation reward if it is sufficiently responsive and correct during the validation period.
+Please note, the last requirement is not applicable for validators who are securing the network from its inception. For securing the Columbus Testnet, it's sufficient to be a market participant in the travel industry.
 
-- The exact amount that a validator must stake is 100,000 CAM
-- The time period one can stake funds for validation is 1 year
+:::
 
-## Can I add more Camino (CAM) tokens to my validator while the staking period is in progress?
+### What are staking parameters for validators on Camino Network?
+
+- The exact amount that a validator must stake is **100,000 CAM**.
+- The time period one can stake funds for a validator is between **2 weeks** (min) and **1 year** (max).
+
+:::note
+
+When a validator completes its role in validating the Camino Mainnet, they will receive their staked Camino tokens back. Additionally, they may also receive a reward for their contribution in securing the network. However, it's worth noting that a validator will only receive a validation reward if it has been sufficiently responsive and accurate during the validation period.
```

Comment:
Thing about reward seems wrong. Our validators receives rewards during their validation period, not at the end of it. And those rewards must be also claimed via tx, they'r not just minted for validators automatically.

Reply:
I think I'll remote that part completely, that part is old and confusing.